docs/src/prototypes/popups/GridImagePicker/GridImagePickerItem.tsx

```diff
@@ -17,19 +16,15 @@ const imageButtonStyles = {
 }
 
 class GridImagePickerItem extends React.Component<GridPickerItemProps> {
-  static defaultProps = {
-    as: 'li',
-  }
-
   render() {
-    const { title, imageSrc, as: ElementType, onClick } = this.props
+    const { title, imageSrc, onClick } = this.props
 
     return (
-      <ElementType>
+      <li>
```

Comment:
This didn't make any reasonable sense, but caused my favourite error, https://github.com/Microsoft/TypeScript/issues/28768